README.md

```diff
@@ -1,3 +1,70 @@
+- **companies** , small and large scale **Agri-businesses** ; for improving the contract farming scheme and enabling sustainability advisers to continuously monitor the overall quality assurance of farmers&#39; (suppliers&#39;) agricultural practices, in order to improve the environmental performance of products, throughout their life-cycle;
+- **policy makers** ; to assist policy makers in determining the performance of crops and crops&#39; varieties under different environments and base decisions on allocating and expanding crops and better planning crop suitability for future climate-resilience.
+
+## Service Template API Overview
+
+This section provides a summary of the Crop Status Monitoring API endpoints:
+
+**Indices List**
+
+- **Description**: Returns a list of all the available service indices.
+- **Input**: No path and/or query parameters or request body required.
+- **Output**: An enumeration of strings, denoting index names or the description of the encountered error (in case something goes wrong) in JSON format.
+
+**Indices Search**
+
+- **Description**: Returns a list of the dates (within a user specified range) for which the requested field growth indices have been calculated, or all requested dates and available indices, in case the service cannot provide such information.
+- **Input**:
+	1. Path parameter: 
+		- the field URN.
+	2. Request body, containing: 
+		- the indices of interest, 
+		- the start date of the interval of interest and
+		- the end date of interest (if omitted, current date is assigned to it).
+- **Output**: A list of the dates for which the requested field growth indices have been calculated or the description of the encountered error (in case something goes wrong) in JSON format.
+
+**Indices Download**
+
+- **Description**: Returns the estimated results for the index of interest at the requested date.
+- **Input**: 
+	1. Path parameters:
+		- the field URN,
+		- the name of the index of interest.
+	2. Query parameter:
+		- the date of interest for indicator retrieval (if omitted, current date is assigned to it).
+- **Output**: The estimated results for the input index at the input date of interest in geotiff format or the description of the encountered error (in case something goes wrong) in JSON format.  
+
+**Satellite Fused Vegetation Index**
+
+- **Description**: Returns the estimated Fused NDVI images for the requested field and time period.
+- **Input**: Request body only, containing:
+	- the field unique reference code,
+	- the start date of the growing season, and 
+	- the end date of the growing season.
+- **Output**: The list of estimated geotiff images in base64 format, along with the correspoding dates of estimation and the sensors that captured the data. In case something goes wrong, the description of the encountered error along with its status code is returned, in JSON format. 
+
+##
```

Comment:
Just a thought (not mandatory requiments): would you recommend standardizing a couple of additional geotif metadata (date, index)?
If so, you could include a "Data Format" section that describes it.